### PR TITLE
streamline sample repo layout

### PR DESCRIPTION
**Pull request by ZhijunZhao**
@zikalino @xscript @aovechki @shizn

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1 @@
-# azure-iot-node-get-started-lesson1
 Azure IoT Node.js sample code for Raspberry Pi 3
```
Comment (ZhijunZhao) [inline README.md]: We need to write up this readme. @shizn

File: package.json [hunk 2]
```diff
@@ -1,15 +1,11 @@
 {
-  "name": "master-cli",
+  "name": "raspberrypi-blink-gulp-script",
```
Comment (zikalino) [inline package.json]: maybe we could make this name more generic, so we could have the same package.json in every sample.

File: package.json [hunk 3]
```diff
@@ -1,15 +1,11 @@
...
   "version": "1.0.0",
-  "description": "Master CLI for Azure IoT Stuff",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "author": "",
-  "license": "ISC",
-  "devDependencies": {    
+  "description": "define gulp tasks used to deploy and run blink code on device",
```
Comment (zikalino) [inline package.json]: Same here, perhaps: "Gulp tasks supporting sample deployment"

File: package.json (hunk 2 above)
Comment (ZhijunZhao) [inline package.json]: agree! let me change it to 'gulp-tasks'

**Pull request merged**